restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -322,6 +333,29 @@ public ResultSet executeSelect(
         .join();
   }
 
+  public ResultSet executeSelect(
+      String keyspace,
+      String collection,
+      List<BuiltCondition> predicates,
+      int pageSize,
+      ByteBuffer pageState)
+      throws UnauthorizedException {
+    // Run generic authorizeDataRead for now
+    getAuthorizationService()
+        .authorizeDataRead(getAuthenticationSubject(), keyspace, collection, SourceAPI.REST);
+    UnaryOperator<Parameters> parametersModifier =
+        p -> ImmutableParameters.builder().pageSize(pageSize).pagingState(pageState).build();
+    return this.builder()
+        .select()
+        .column(DocumentDB.allColumns())
+        .writeTimeColumn("leaf")
```

Comment:
As far as I can see, the leaf column type is text. Why the `writeTime` is text?

Reply:
`writeTime` is a long. This query is pulling the writeTime of the leaf column in order to return the most recent version I believe.